script.js

```diff
@@ -143,13 +151,75 @@ const confirmOfClearForm = () => {
     });
 };
 
+const valuesToForm = id => {
+  const values = $$("filmsDataTable").getItem(id);
+  $$("addFilmForm").setValues(values);
+};
+
+webix.DataStore.prototype.sorting.as.sortByFloat = (first, second) => {
```

Comment:
now you have correct numbers in the data. you can use sort:"int" because it has the same logic

Reply:
Fixed it. Ready to review.